packages/react/src/components/ExpandCollapsePanel/ExpandCollapsePanel.tsx

```diff
@@ -13,7 +13,7 @@ export interface ExpandCollapsePanelProps
   open?: boolean;
   children: React.ReactNode;
   animationTiming?: number | boolean;
-  onToggle: (e: React.MouseEvent<HTMLButtonElement>) => void;
+  onToggle?: (e: React.MouseEvent<HTMLButtonElement>) => void;
```

Comment:
Why is `onToggle` optional? It should be implicitly required.

Reply:
Is it currently being required? Because you can definitely do an `ExpandCollapse` without passing in `onToggle` which seems to not cause issues since it is being set with a `defaultProp`. 

However, with the `Accordion` component, it is throwing a TypeScript error if you do not explicitly pass in the `onToggle` (below is the error when trying it as not being optional). I guess an alternative solution would be to always pass in the `onToggle` in the accordion component itself? 

`TS2741: Property 'onToggle' is missing in type '{ children: Element[]; }' but required in type 'Pick<AccordionProps, "slot" | "style" | "title" | "children" | "className" | "aria-label" | "defaultChecked" | "defaultValue" | "suppressContentEditableWarning" | ... 246 more ... | "animationTiming">'.`

Reply:
@scurker Just pinging in case you did not see the above